### PR TITLE
Ethhack2019

### PR DESCRIPTION
**Pull request by deeayeen**
##### Description

Add ethhack 2019 landing page.

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [x] Added relevant tests
- [x] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [x] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [x] Completed: Code review by core team and any requested changes

##### Refers/Fixes

[3958](https://github.com/gitcoinco/web/issues/3958)

###### Contributor

- [x] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Tested all changes **locally**
- [x] Verified existing functionality
- [x] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/marketing/views.py [hunk 1]
```diff
@@ -724,7 +724,7 @@ def leaderboard(request, key=''):
 
     if amount:
         amount_max = amount[0][0]
-        top_earners = ranks.order_by('-amount')[0:3].values_list('github_username', flat=True)
+        top_earners = ranks.order_by('-amount')[0:5].values_list('github_username', flat=True)
```
Comment (octavioamu) [inline app/marketing/views.py]: Are this changes related?

Comment (octavioamu) [reply app/marketing/views.py]: think this PR is not going to be merged at the same time as leaderboard changes, so all content related to leaderboard need to be removed from this PR to avoid break things when deployed.

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+{% load i18n static %}
+<html lang="en">
+<head>
+  {% include 'shared/head.html' %}
+  <link rel="stylesheet" href="{% static "v2/css/dashboard.css" %}">
+  <link rel="stylesheet" href="{% static "v2/css/ethhack2019.css" %}">
+  <meta charset="UTF-8">
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: this content is already in head.html

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <div class="col-3">
+          <img class="img-fluid align-middle" style="width: 100%" src="{% static "v2/images/hackathon/consensys.png" %}" />
+        </div>
+        <div class="col-3">
+          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/ethmain.svg" %}" />
+        </div>
+      </div>
+    </section>
+
+    <script src="{% static "v2/js/bootstrap.min.js" %}"></script>
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: if you are using bootstrap.js you need to also call pooper.js or is going to fail

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+<head>
+  {% include 'shared/head.html' %}
+  <link rel="stylesheet" href="{% static "v2/css/dashboard.css" %}">
+  <link rel="stylesheet" href="{% static "v2/css/ethhack2019.css" %}">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Eth Hackathon 2019</title>
+</head>
+  <body class="text-center" style="background-color: #180944; color: white">
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: please take a look at all the utilities classes you have from bootstrap https://getbootstrap.com/docs/4.3/utilities/spacing/

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+      <a class="btn font-weight-bold mr-2 btns" style="background-color: #f9006c; border-radius: 30px; width: 100px; font-size: .4rem">Register</a>
+      <a class="btn font-weight-bold ml-2 btns" style="background-color: #3e00ff; border-radius: 30px; width: 100px; font-size: .4rem">Fund a Prize</a>
+    </div>
+
+    <section class="container-fluid header" style="overflow: visible; background-image: url(/static/v2/images/hackathon/ethback.jpg);">
+      <h2 class="font-weight-bolder part" style="color: white; padding-top: 3rem; font-size: 1.2rem; text-decoration: underline; text-underline-position: under;">April 1st &#8212; 30th</h2>
+      <h2 class="pdesc3" style="color: white; padding-top: 1.25rem; padding-left: 3rem; padding-right: 3rem; padding-bottom: 3rem; font-size: 0.6rem">Hackathon challenges will be posted as bounties, with the best hacks receiving prizes in ETH & ERC-20 tokens. Main track winners will receive free tickets to Ethereal NY to present their project live on stage.</h2>
+    </section>
+
+    <section style="background: white">
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: bg-white

File: app/marketing/views.py (hunk 1 above)
Comment (deeayeen) [reply app/marketing/views.py]: fixed

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html (hunk 3 above)
Comment (deeayeen) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: fixed

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html (hunk 5 above)
Comment (deeayeen) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: fixed

**Comment by octavioamu** [comment]
to center elements for example instead of position absolute you can take a look how it was used here 
https://github.com/gitcoinco/web/blob/master/app/dashboard/templates/social_contribution_modal.html (go to a bounty and press share to see it alive) 
Or even better coded check kudos landing https://gitcoin.co/kudos/

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html (hunk 4 above)
Comment (deeayeen) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: fixed

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html (hunk 2 above)
Comment (deeayeen) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: fixed

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          <img class="img-fluid align-middle" style="width: 100%" src="{% static "v2/images/hackathon/consensys.png" %}" />
+        </div>
+        <div class="col-3">
+          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/ethmain.svg" %}" />
+        </div>
+      </div>
+    </section>
+
+    <script src="{% static "v2/js/bootstrap.min.js" %}"></script>
+    <script src="{% static "v2/js/popper.min.js" %}"></script>
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: as pooper is a dependency needs to come first

Comment (deeayeen) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: thanks fixed

Comment (octavioamu) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: but anyway are you using some js from bootstrap here?

Comment (octavioamu) [reply app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: if not you don't need to put it here

**Comment by octavioamu** [comment]
Thanks @deeayeen finally can you add the links?
Here is the hacker link https://gitcoin.typeform.com/to/j7CSbV
And sponsor https://gitcoin.typeform.com/to/YeIflB
